foundations/javascript_basics/problem_solving.md

```diff
@@ -110,6 +110,8 @@ The above code should create a little popup box that asks the user for a number.
 
 With that done, let's move on to the next subproblem: "Loop from 1 to the entered number". There are many ways to do this in JavaScript. One of the common ways - that you actually see in many other languages like Java, C++, and Ruby - is with the for-loop:
 
+You can read more about for loops and how they're used in this MDN article: [For Loops | MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for).
```

Comment:
Not against adding a link for the for loop. That said given it's explained already a fair bit slightly below the code. I would rather it be linked at the the "for-loop" part, rather than adding another line about specifically reading more about it. If you would put the link on line 111 on the "for-loop" text I would be happy to merge it.